jira/client.py

```diff
@@ -1660,9 +1660,19 @@ def _get_user_id(self, user: str) -> str:
         try:
             user_obj: User
             if self._is_cloud:
-                user_obj = self.search_users(query=user, maxResults=1)[0]
+                users = self.search_users(query=user)
             else:
-                user_obj = self.search_users(user=user, maxResults=1)[0]
```

Comment:
I'm a bit concerned about removing the `maxResults`, can we set it to a reasonable default still?

Reply:
Reset as 20 - this should be reasonable still? @adehad

Reply:
yep